releases/7.6.0.md

```diff
@@ -301,12 +355,11 @@ Minor incompatibilities:
 
     # Only handle i18n routing in active project. This will still handle the routes provided by Arches core and Arches applications,
     # but handling i18n routes in multiple places causes application errors.
-    if settings.APP_NAME != "Arches" and settings.APP_NAME not in settings.ARCHES_APPLICATIONS:
+    if settings.APP_NAME != "Arches" and settings.APP_NAME not in arches_applications():
```

Comment:
This logic will need reworking I think -- `settings.APP_NAME` is in `INSTALLED_APPS`, and I'm assuming the default for all Arches projects will be that `is_arches_application` is true.

Reply:
Yeah, this is (one reason) why I wasn't assuming all projects are "arches applications" -- we'll need to find out if there are any blockers to collapsing that distinction.

Reply:
Now I just check if the project is last in INSTALLED_APPS. Open to ideas.